src/main/java/InputDto.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+
+public class InputDto {
+    private List<String> names;
+    private int numberOfAttempts;
+
+    public InputDto(String names, String numberOfAttempts) {
+        if (isBlank(names)) throw new IllegalArgumentException("공백은 입력할 수 없습니다.");
+        this.names = Arrays.asList(names.split(","));
+        this.numberOfAttempts = toInteger(numberOfAttempts);
+    }
+
+    public static int toInteger(String value){
+        try {
+            return Integer.parseInt(value);
+        } catch (NumberFormatException ex) {
+            throw new IllegalArgumentException("정수만 입력이 가능합니다.");
+        }
+    }
+
+    public static boolean isBlank(String x) {
+        return "".equals(x) || " ".equals(x);
```

Comment:
```suggestion
        return "".equals(x.trim());
```
이렇게 표현할 수 있습니다!
공백 또한 상수로 만들어주면 어떨까요?